src/Tuura/Concept/STG/Circuit.hs

```diff
@@ -96,7 +96,6 @@ handshake11 a b = handshake a b <> initialise a True <> initialise b True
 
 me :: Eq a => a -> a -> CircuitConcept a
```

Comment:
Let's add a note: `TODO: Restrict the initial state so that a=b=1 is not allowed.`